playwright-e2e/helpers/e2eJourneyHelper.ts

```diff
@@ -13,6 +13,17 @@ export async function citizenAdoptionSignInWithNoPartner(
   await app.basePage.clickSaveAndContinue();
 }
 
+export async function citizenAdoptionSignInSingleParent(
+  app: App,
```

Comment:
Not sure I see the point of the citizenAdoptionSignInSingleParent function as it's identical to citizenAdoptionSignInWithPartner.  Can't you just combine them both into a generic function?

Reply:
That is the eventual plan. I will be utilising it with the LA journey and see how far I can go with refactoring the e2e tests in Playwright. Once I know that and get it working, I will come back to the e2eJourneyHelper.ts and refactor it so it's less procedural and more Object-Oriented. e.g. citizenAdoptionSignInSingleParent should be related to the SignIn page Object

Reply:
why not just use citizenAdoptionSignInWithPartner in the meantime then? Rather than create a function you know you're going to delete in the future.

Reply:
I'll have to rename the function so that the test code does not become misled to what it is actually doing, it will have to be renamed to something more generic as suggested. If It is okay in the time being I can make it generic in a procedural sense and then make it Object Oriented at a later Sprint stage?

Reply:
renaming the function to something more generic seems like a good compromise. That way you're not creating more work/risk for in the future (when you/someone has to remove the duplicate function)

Reply:
I have renamed the two functions in question to a single generic named function for the time being.